haystack/nodes/answer_generator/openai.py

```diff
@@ -52,25 +53,29 @@ class OpenAIAnswerGenerator(BaseGenerator):
     def __init__(
         self,
         api_key: str,
-        model: str = "text-curie-001",
-        max_tokens: int = 13,
+        model: str = "text-davinci-003",
+        max_tokens: int = 50,
```

Comment:
I'd also increase the default here as davinci 003 has a much bigger max seq length of 4000 tokens and the generated answers are often quite verbose